ckanext/issues/tests/controllers/test_spam.py

```diff
@@ -41,7 +41,7 @@ def test_abuse_label_appears_for_publisher(self):
         )
         res_chunks = parse_issues_show(response)
         assert_in('Test Issue', res_chunks['issue_name'])
-        assert_in('Abuse', res_chunks['issue_comment_label'])
+        assert_in('Hidden from normal users', res_chunks['issue_comment_label'])
```

Comment:
There should also be a message saying it is abuse. Can you test for that (in addition)?